### PR TITLE
add object property labels

### PR DESCRIPTION
**Pull request by stevenchong**
@mpsaloha had asked me to add labels to the object properties in OBOE-Core. Also, the order of some of the lines changed. Not sure if this a Protege thing, but I've come across this issue before in the ENVO group.

### CODE REVIEW

**Comment by stevenchong** [comment]
@mbjones @amoeba Please take a look at this PR and let me know if any changes are needed. Thanks

File: oboe-core.owl [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
-     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:swrl="http://www.w3.org/2003/11/swrl#"
+     xmlns:swrlb="http://www.w3.org/2003/11/swrlb#"
+     xmlns:owl2xml="http://www.w3.org/2006/12/owl2-xml#"
+     xmlns:oboe-core="http://ecoinformatics.org/oboe/oboe.1.2/oboe-core.owl#">
     <owl:Ontology rdf:about="http://ecoinformatics.org/oboe/oboe.1.2/oboe-core.owl">
         <cc:license rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">http://creativecommons.org/licenses/by/3.0/</cc:license>
-        <rdfs:label>oboe-core</rdfs:label>
+        <rdfs:comment>Copyright (c) 2006-2016 The Regents of the University of California.  All rights reserved. This work is licensed under the Creative Commons Attribution 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/ or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View, California, 94041, USA.</rdfs:comment>
```
Comment (mbjones) [inline oboe-core.owl]: Update dates to 2006-2019

**Comment by mbjones** [comment]
See my comment above, but a quick review looks good to me.

**Comment by amoeba** [comment]
LGTM, thanks @stevenchong. Let's just wait for @mpsaloha 	and then merge.

**Comment by stevenchong** [comment]
FYI, he's out of town this week for a workshop so I say we should merge if we don't hear from him by Thursday morning (assuming there's no edits needed).

**Comment by amoeba** [comment]
Got it, let's do that. Thanks!

**Review by mpsaloha — COMMENTED** [review]
This looks good to me, thanks Steven.
Bryce, can you accept and upload copy to Bioportal for dereferencing our annotation predicates?
thanks!

**Comment by mbjones** [comment]
First the PR needs to get merged, and then the release would need to be
merged and tagged.  Then it can go to bioportal.

Matt

**Comment by amoeba** [comment]
Thanks all. @mpsaloha 	yep, 'll get that process going.

**Pull request merged**